Makefile

```diff
@@ -7,7 +7,7 @@ VERSION=0.4.1
 OS_ARCH=$(shell go env GOOS)_$(shell go env GOARCH)
 PLUGIN_DIR=~/.terraform.d/plugins
 DOCGEN_RESOURCES_DIR=docgen-resources
-
+GOPRIVATE="github.com/atko-pam,github.com/okta"
```

Comment:
do we need to update the makefile with these or was this locally added to test?

Reply:
No this was suggested by Evan to easily update the pam sudo sdk version and other dependencies.